pkgs/by-name/sc/sccmhunter/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "sccmhunter";
+  version = "1.0.6-unstable-2024-10-30";
+  pythonRelaxDeps = true;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "garrettfoster13";
+    repo = "sccmhunter";
+    rev = "7f493d1499fec1a694d949d9de17f25acdb6a0a9";
+    hash = "sha256-rMkm4SDIiPrlNJWlnwD2td7yzkTNlTzZ3KaiKK5TOfg=";
+  };
+
```

```suggestion

  build-system = with python3Packages; [
    setuptools
  ];
  
```
upstream uses the `setuptools` to build the package